app/assets/javascripts/provider/admin/apiconfig/services/service_discovery.js.coffee

```diff
@@ -14,13 +14,13 @@ $ ->
 
   show_service_form = (source, discover_func, scratch_func)->
     if source == 'discover'
-      form_scratch.classList.add('is-hidden')
-      form_discover.classList.remove('is-hidden')
+      $(form_scratch).addClass 'is-hidden'
```

Comment:
I see you decided to keep it jQuery-ish. Why? Can `classList.add|remove` fail in any supported browser?